python/py-attrs/Portfile

```diff
@@ -11,7 +11,7 @@ platforms           darwin
 supported_archs     noarch
 license             MIT
 
-python.versions     27 35 36 37 38 39 310
+python.versions     27 35 36 37 38 39 310 311
 
 maintainers         {khindenburg @kurthindenburg} openmaintainer
 
```

Comment:
regarding `py-six` that is added here to `depends_test` - where does that come from, I see no trace of `six` anywhere in upstream's repository

Reply:
Indeed it doesn't required. Removed.